### DISCUSSION TITLE
【潜水】远景爆破案即将开启

### DISCUSSION THREAD

**Discussion by liu1145** [share121/inter-knot#1495] [其他]
![屏幕截图 2024-08-11 204014](https://github.com/user-attachments/assets/53898b00-98e9-482a-a704-4f1275e03b6a)
新艾利都律法院现已发出公告，轰动一时的【远景爆破案】已确定第一次开庭时间。审理前日，包括被告（珀尔曼），诉讼代理人（万能事务所狡兔屋）等在内的相关人员，将依照惯例，乘坐专用空艇前往最高裁决候审院。
据悉，本案将采用公开审理的形式，共开放300个旁听席位，已经成年并具有新艾利都市民身份的人员，均可申请旁听。此外，包括丽都频道在内的8家媒体，已表示将对此案审理过程进行现场直播。

**Comment by reversed-PAI** [comment]
上个月的事儿了，这新闻有点滞后啊，话说有人知道珀尔曼藏哪儿了吗？治安局给那么高的悬赏还没抓到吗？

**Comment by EhuanyingK** [comment]
cy

**Comment by CocoonedSoul** [comment]
给绫波零
